content/news/2021-08-19-gsoc-coverage.md

```diff
@@ -0,0 +1,131 @@
+# Ruma API Coverage GSoC Coming to an End
```

Comment:
Please add TOML frontmatter with a `title` (removing this heading) and `author = "Adam Blanchet"`. TOML frontmatter is enclosed in two lines that are just three `+` signs. You can check the other files in this directory and/or install [Zola](https://www.getzola.org/documentation/getting-started/overview/) and run `zola serve` locally to check out what it looks like.

I think I'd also like to kinda sync the titles of the two end-of-gsoc blog posts, let's discuss that on Matrix.